gui/markdown-viewer.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Cozy Drive</title>
+    <link rel="stylesheet" href="app.css">
```

Comment:
how this css file is generated? Is it the preexisting one so we're loading all the CSS (https://github.com/cozy-labs/cozy-desktop/blob/86b7e6c230ace0cc6d273a0dc209380ff057bd13/gui/styles/app.styl) of the desktop app? 

Since in `_markdown_viewer.styl` I don't see any import, it does mean that this file is pretty standalone?

Reply:
I'm actually relying on the style of the rest of the app. So I need to import `app.css`.